experimental/traffic-portal/package.json

```diff
@@ -1,119 +1,119 @@
-    "dev:ssr": "ng run traffic-portal:serve-ssr",
-    "serve:ssr": "node dist/traffic-portal/server/main.js",
-    "build:ssr": "ng build --configuration production && ng run traffic-portal:server:production",
-    "prerender": "ng run traffic-portal:prerender"
-  },
-  "private": true,
-  "dependencies": {
-    "@angular/animations": "^14.0.2",
-    "@angular/cdk": "^14.0.2",
-    "@angular/common": "^14.0.2",
-    "@angular/compiler": "^14.0.2",
-    "@angular/core": "^14.0.2",
-    "@angular/forms": "^14.0.2",
-    "@angular/material": "^14.0.2",
-    "@angular/platform-browser": "^14.0.2",
-    "@angular/platform-browser-dynamic": "^14.0.2",
-    "@angular/platform-server": "^14.0.2",
-    "@angular/router": "^14.0.2",
-    "@fortawesome/angular-fontawesome": "^0.10.0",
-    "@fortawesome/fontawesome-svg-core": "^6.1.0",
-    "@fortawesome/free-regular-svg-icons": "^6.0.0",
-    "@fortawesome/free-solid-svg-icons": "^6.0.0",
-    "@nguniversal/express-engine": "^14.0.1",
-    "ag-grid-angular": "^27.3.0",
-    "ag-grid-community": "^27.3.0",
-    "argparse": "^2.0.1",
-    "chart.js": "^2.9.4",
-    "express": "^4.15.2",
-    "rxjs": "~6.6.0",
-    "tslib": "^2.0.0",
-    "zone.js": "~0.11.4"
-  },
-  "devDependencies": {
-    "@angular-devkit/build-angular": "^14.0.2",
-    "@angular-eslint/builder": "14.0.0-alpha.3",
-    "@angular-eslint/eslint-plugin": "14.0.0-alpha.3",
-    "@angular-eslint/eslint-plugin-template": "14.0.0-alpha.3",
-    "@angular-eslint/schematics": "14.0.0-alpha.3",
-    "@angular-eslint/template-parser": "14.0.0-alpha.3",
-    "@angular/cli": "^14.0.2",
-    "@angular/compiler-cli": "^14.0.2",
-    "@nightwatch/schematics": "^0.1.9",
-    "@types/argparse": "^2.0.2",
-    "@types/chart.js": "^2.9.34",
-    "@types/express": "^4.17.0",
-    "@types/jasmine": "~3.6.0",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/mocha": "^9.1.1",
-    "@types/nightwatch": "2.0.1",
-    "@types/node": "^14.17.34",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "axios": "^0.27.2",
-    "chromedriver": "^106.0.1",
-    "codelyzer": "^6.0.0",
-    "eslint": "^8.2.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jsdoc": "^37.0.3",
-    "eslint-plugin-prefer-arrow": "^1.2.3",
-    "geckodriver": "3.0.2",
-    "jasmine-core": "^3.10.1",
-    "karma": "~6.3.2",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage": "~2.2.0",
-    "karma-jasmine": "~4.0.0",
-    "karma-jasmine-html-reporter": "^1.5.0",
-    "nightwatch": "2.0.0-beta.3",
-    "trafficops-types": "^3.1.0-beta-16",
-    "ts-node": "~8.3.0",
-    "typescript": "^4.5.4"
-  },
-  "optionalDependencies": {
-    "@compodoc/compodoc": "^1.1.18"
-  }
+	"name": "traffic-portal",
+	"version": "0.4.1",
+	"description": "The web UI for interacting with the Apache Traffic Control CDN management and operation system.",
+	"homepage": "https://github.com/apache/trafficcontrol",
```

Comment:
Doesn't `ng update` overwrite this to spaces? I think that's why I didn't make it tabs to begin with. If that's not/no longer true, then I'm good with the change, just would like to be sure.